lib/datadog/tracing/contrib/sidekiq/server_tracer.rb

```diff
@@ -50,27 +48,21 @@ def call(worker, job, queue)
               span.set_tag(Ext::TAG_JOB_QUEUE, job['queue'])
               span.set_tag(Ext::TAG_JOB_WRAPPER, job['class']) if job['wrapped']
               span.set_tag(Ext::TAG_JOB_DELAY, 1000.0 * (Time.now.utc.to_f - job['enqueued_at'].to_f))
-              span.set_tag(Ext::TAG_JOB_ARGS, job['args']) if tag_args && !job['args'].nil? && !job['args'].empty?
+              span.set_tag(Ext::TAG_JOB_ARGS, quantize_args(job['args'])) if !job['args'].nil? && !job['args'].empty?
 
               yield
             end
           end
 
           private
 
-          def configuration
-            Datadog.configuration.tracing[:sidekiq]
+          def quantize_args(args)
+            quantize_options = configuration[:quantize][:args] || {}
+            Contrib::Utils::Quantization::Hash.format(args, quantize_options)
           end
 
-          def worker_config(resource, key)
```

Comment:
@dudo any particular reason to remove Worker-level configuration?

Reply:
It seems as though it was only there as a convenience method for the tests. There are no docs for it... no reference anywhere.

Lemme know if it's "real", and I'll put it back, and add something to the docs.

Reply:
Keeping it for now I think is the best option.

Reply:
So... While putting this back, I noticed a few things that support my initial concerns. This is a private method used by this middleware. The only flags it's trying to grab are `service_name` and `tag_args`. If we really want to support dynamically setting the service, feels like we should leverage `Datadog.tracer.active_span`, no? Feels a bit more idiomatic. Based on the specs I fixed, it really feels like this was just a test helper function.

Again... no documentation about this anywhere - I'd really advocate we lose it.